cabal.project

```diff
@@ -417,11 +417,12 @@ source-repository-package
   subdir: http-client
 
 constraints:
-  ip < 1.5,
-  hedgehog >= 1.0,
-  bimap >= 0.4.0,
-  brick >= 0.47,
-  libsystemd-journal >= 1.4.4
+    ip < 1.5
+  , hedgehog >= 1.0
+  , bimap >= 0.4.0
+  , brick >= 0.47
+  , libsystemd-journal >= 1.4.4
+  , systemd >= 2.3.0
```

Comment:
Does this branch need to be rebased? I see to remember doing exactly this just recently.

Reply:
Done.